content/en/docs/reference/glossary/kube-scheduler.md

```diff
@@ -4,15 +4,15 @@ id: kube-scheduler
 date: 2018-04-12
 full_link: /docs/reference/command-line-tools-reference/kube-scheduler/
 short_description: >
-  Control plane component that watches for newly created pods with no assigned node, and selects a node for them to run on.
+  Control plane component that watches for newly created Pods that have not yet been assigned to any nodes, and selects a node for them to run on.
 
 aka: 
 tags:
 - architecture
 ---
 Control plane component that watches for newly created
-{{< glossary_tooltip term_id="pod" text="Pods" >}} with no assigned
-{{< glossary_tooltip term_id="node" text="node">}}, and selects a node for them
+{{< glossary_tooltip term_id="pod" text="Pods" >}} that have not yet been assigned to any
+{{< glossary_tooltip term_id="node" text="nodes">}}, and selects a node for them
```

Comment:
The text is not as confusing as it sounds like if ready carefully. If we go by this mean, I think we'll come across multiple such texts for which no issue/PR has ever been raised (unless that's critical).